digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/masstransitstop/MassTransitStopService.scala

```diff
@@ -50,6 +49,8 @@ trait AbstractPublishInfo {
 
 case class PublishInfo(asset: Option[PersistedMassTransitStop]) extends AbstractPublishInfo
 
+sealed case class InvalidParameterException (message:String) extends IllegalArgumentException(message)
```

Comment:
En ehkä ymmärrä, mitä lisäarvoa tämä rakenne tuo, mutta ton niminen poikkeus löytyy jo valmiiksi java.security -kirjastosta, mikä voi olla hämäävää. Onko tuossa illegalArgumentExceptionissa sellaisenaan vikaa?

Reply:
joo niin on mut en halunnut sekoitaa tätä siihen. Se tais liityyä johonkin tietoturva hommaan.

Reply:
[InvalidParameterException](https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/security/InvalidParameterException.html)
This exception, designed for use by the JCA/JCE engine classes, is thrown when an invalid parameter is passed to a method.

Reply:
IllegalArgumentException on geneerinen. Haluan olla tarkka mistä virhe johtuu.

Reply:
Ok. Luin ton dokumentaation kans. Samannimisyys hämmensi, mutta käy hyvin, jos sun mielestä parantaa virheen käsittelyä.